README.md

```diff
@@ -54,11 +54,13 @@ Please install these tools using pip:
 
     pip install gftools
 
+(Mac OS users on M1 chips will need to upgrade pip first to support universal wheels: `pip install -u pip`.)
```

Comment:
Is it lowercase `-u` on Mac?

I'm only asking this because, at least on GNU+Linux, there's no `-u`. The correct way is uppercase `-U` and I would expect the flag to be the same on all systems.

Reply:
Oops, you're right. I was doing it from memory!